velero-plugins/main.go

```diff
@@ -155,3 +157,14 @@ func newImageStreamRestorePlugin(logger logrus.FieldLogger) (interface{}, error)
 func newImageStreamTagRestorePlugin(logger logrus.FieldLogger) (interface{}, error) {
 	return &migimagestreamtag.RestorePlugin{Log: logger}, nil
 }
+
+func newServiceAccountBackupPlugin(logger logrus.FieldLogger) (interface{}, error) {
+	saBackupPlugin := &migsa.BackupPlugin{Log: logger}
+
+	err := saBackupPlugin.InitSCCMap()
```

Comment:
My only concern with this is that we are only establishing the SCC map when the velero plugin is registered. If any SCC changes are made after the velero pod starts up then we will be working with stale data. Perhaps in the plugin itself we have an `UpdateSCCMap()` call that refreshes the data?

I don't consider this a blocker for merging this PR because I don't see this being a huge problem. SCCs aren't touched very often.

Reply:
Is there an easy way to make this update for each backup without having to regenerate it for each service account?

Reply:
I can try to set up a watcher and update SCCMap when needed